service/frontend/wrappers/ratelimited/api_generated.go

```diff
@@ -0,0 +1,718 @@
+	if rp1 == nil {
+		err = validate.ErrRequestNotSet
+		return
+	}
+	if rp1.GetDomain() == "" {
+		err = validate.ErrDomainNotSet
+		return
+	}
+	// Count the request in the host RPS,
+	// but we still accept it even if RPS is exceeded
+	h.allowDomain(ratelimitTypeWorker, rp1.GetDomain())
+	return h.wrapped.RecordActivityTaskHeartbeatByID(ctx, rp1)
+}
+
+func (h *apiHandler) RefreshWorkflowTasks(ctx context.Context, rp1 *types.RefreshWorkflowTasksRequest) (err error) {
+	if rp1 == nil {
+		err = validate.ErrRequestNotSet
+		return
+	}
+	if rp1.GetDomain() == "" {
+		err = validate.ErrDomainNotSet
+		return
+	}
+	if ok := h.allowDomain(ratelimitTypeUser, rp1.GetDomain()); !ok {
+		err = &types.ServiceBusyError{Message: "Too many outstanding requests to the cadence service"}
+		return
+	}
+	return h.wrapped.RefreshWorkflowTasks(ctx, rp1)
+}
+
+func (h *apiHandler) RegisterDomain(ctx context.Context, rp1 *types.RegisterDomainRequest) (err error) {
+	return h.wrapped.RegisterDomain(ctx, rp1)
+}
+
+func (h *apiHandler) RequestCancelWorkflowExecution(ctx context.Context, rp1 *types.RequestCancelWorkflowExecutionRequest) (err error) {
+	if rp1 == nil {
+		err = validate.ErrRequestNotSet
+		return
+	}
+	if rp1.GetDomain() == "" {
+		err = validate.ErrDomainNotSet
+		return
+	}
+	if ok := h.allowDomain(ratelimitTypeUser, rp1.GetDomain()); !ok {
+		err = &types.ServiceBusyError{Message: "Too many outstanding requests to the cadence service"}
+		return
+	}
+	return h.wrapped.RequestCancelWorkflowExecution(ctx, rp1)
+}
+
+func (h *apiHandler) ResetStickyTaskList(ctx context.Context, rp1 *types.ResetStickyTaskListRequest) (rp2 *types.ResetStickyTaskListResponse, err error) {
+	if rp1 == nil {
+		err = validate.ErrRequestNotSet
+		return
+	}
+	if rp1.GetDomain() == "" {
+		err = validate.ErrDomainNotSet
+		return
+	}
+	// Count the request in the host RPS,
+	// but we still accept it even if RPS is exceeded
+	h.allowDomain(ratelimitTypeWorker, rp1.GetDomain())
+	return h.wrapped.ResetStickyTaskList(ctx, rp1)
+}
+
+func (h *apiHandler) ResetWorkflowExecution(ctx context.Context, rp1 *types.ResetWorkflowExecutionRequest) (rp2 *types.ResetWorkflowExecutionResponse, err error) {
+	if rp1 == nil {
+		err = validate.ErrRequestNotSet
+		return
+	}
+	if rp1.GetDomain() == "" {
+		err = validate.ErrDomainNotSet
+		return
+	}
+	if ok := h.allowDomain(ratelimitTypeUser, rp1.GetDomain()); !ok {
+		err = &types.ServiceBusyError{Message: "Too many outstanding requests to the cadence service"}
+		return
+	}
+	return h.wrapped.ResetWorkflowExecution(ctx, rp1)
+}
+
+func (h *apiHandler) RespondActivityTaskCanceled(ctx context.Context, rp1 *types.RespondActivityTaskCanceledRequest) (err error) {
+	if rp1 == nil {
+		err = validate.ErrRequestNotSet
+		return
+	}
+	if rp1.TaskToken == nil {
+		err = validate.ErrTaskTokenNotSet
+		return
+	}
```

Comment:
at some point we should probably make some validation-wrappers that occur early and get rid of this need at random levels...
but 👍 yep, same as existing behavior, LGTM